### PR TITLE
443 for TLS

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

**Review by davidzhao — APPROVED** [review]

File: pkg/service/roommanager.go [hunk 1]
```diff
@@ -595,7 +595,7 @@ func (r *RoomManager) iceServersForRoom(ri *livekit.Room, tlsOnly bool) []*livek
 			urls = append(urls, fmt.Sprintf("turn:%s:%d?transport=udp", r.config.RTC.NodeIP, r.config.TURN.UDPPort))
 		}
 		if r.config.TURN.TLSPort > 0 {
-			urls = append(urls, fmt.Sprintf("turns:%s:%d?transport=tcp", r.config.TURN.Domain, r.config.TURN.TLSPort))
```
Comment (cnderrauber) [inline pkg/service/roommanager.go:598]: If tls not listen 443, client will failed

File: pkg/rtc/participant.go [hunk 2]
```diff
@@ -1039,11 +1039,14 @@ func (p *ParticipantImpl) setupTransportManager() error {
...
+		if p.params.ClientConf == nil {
+			p.params.ClientConf = &livekit.ClientConfiguration{}
+		}
 		if iceConfig.PreferSub == types.PreferTls {
-			if p.params.ClientConf == nil {
-				p.params.ClientConf = &livekit.ClientConfiguration{}
-			}
 			p.params.ClientConf.ForceRelay = livekit.ClientConfigSetting_ENABLED
+		} else {
+			p.params.ClientConf.ForceRelay = livekit.ClientConfigSetting_DISABLED
```
Comment (cnderrauber) [inline pkg/rtc/participant.go:1049]: Do we need disable force relay?

File: pkg/service/roommanager.go (hunk 1 above)
Comment (boks1971) [reply pkg/service/roommanager.go:598]: Yes, external facing TURN should have a listener at 443. Else, thing will fail.

File: pkg/rtc/participant.go (hunk 2 above)
Comment (boks1971) [reply pkg/rtc/participant.go:1049]: Just a safety. We are just disabling forcing the relay. Relay will still be used in normal sequence.

**Pull request merged**